packages/genesys-spark-components/src/components/stable/gux-form-field/functional-components/gux-form-field-legend-label/gux-form-field-legend-label.tsx

```diff
@@ -1,30 +1,26 @@
 import { FunctionalComponent, h, VNode } from '@stencil/core';
 
-import { GuxFormFieldLabelPosition } from '../../gux-form-field.types';
-
 interface GuxFormFieldLegendLabelProps {
-  position: GuxFormFieldLabelPosition;
   required: boolean;
   labelText: string;
 }
 
 export const GuxFormFieldLegendLabel: FunctionalComponent<
   GuxFormFieldLegendLabelProps
-> = ({ position, required, labelText }, children): VNode[] => {
+> = ({ required, labelText }, children): VNode[] => {
   return [
     <legend class="gux-form-field-legend-label gux-screenreader">
       {children}
     </legend>,
-    <span
+    <div
```

Comment:
This change is just to make the spacing between legend label items work correctly.